.github/workflows/code-static-analysis.yml

```diff
@@ -0,0 +1,77 @@
+# Runs the CodeQL static code analysis
+
+name: "Static Code Analysis"
+
+on:
+  push:
+    paths:
+      - '**.js'
+      - '**.py'
+      - '**.html'
+  pull_request:
+    paths:
+      - '**.js'
+      - '**.py'
+      - '**.html'
+  schedule:
+    #        ┌───────────── minute (0 - 59)
+    #        │ ┌───────────── hour (0 - 23)
+    #        │ │ ┌───────────── day of the month (1 - 31)
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        * * * * *
+    - cron: '30 1 * * 0'
+  workflow_dispatch:
+
+jobs:
+  CodeQL-Build:
+    # CodeQL runs on ubuntu-latest, windows-latest, and macos-latest
+    runs-on: ubuntu-latest
+    # Skip duplicate job for local branches
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
I am wondering, will this condition allow `schedule` and `workflow_dispatch` events?

Reply:
Good spot. Fixed.